WpfMath/TexFormulaParser.cs

```diff
@@ -571,8 +574,8 @@ private Atom ConvertCharacter(TexFormula formula, string value, ref int position
             if (IsSymbol(character))
             {
                 // Character is symbol.
-                var symbolName = symbols[character];
-                if (symbolName == null)
+                var symbolName = symbols.ElementAtOrDefault(character);

```

Comment:
Here you've fixed the bug I don't know about. Please add a unit test for that bug if that's possible.